domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/StatementOfMeans.java

```diff
@@ -0,0 +1,113 @@
+package uk.gov.hmcts.cmc.domain.models.statementofmeans;
+
+import lombok.Builder;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static java.util.Collections.emptyList;
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@Builder
+public class StatementOfMeans {
+    private final ResidenceType residenceType;
+    private final Dependant dependant;
+    private final Employment employment;
+    private final List<BankAccount> bankAccounts;
+    private final List<Debt> debts;
+    private final List<Income> incomes;
+    private final List<Expense> expenses;
```

Comment:
In screen `What are your regular expenses?` (https://hmcts-cmc-prototype.herokuapp.com/prototypes/prototype-jun-2018/defendant/task-list/owe-all/statement/expenses) type `other` is also structured differently (see above).

Reply:
if selected others, the string entered by user will be used as type. May be like `Others: General expense`